aci/resource_aci_cloudctxprofile.go

```diff
@@ -212,6 +230,12 @@ func resourceAciCloudContextProfileCreate(ctx context.Context, d *schema.Resourc
 		cloudCtxProfileAttr.Type = Type.(string)
 	}
 
+	cloudBrownfield := ""
+	if tmpVar, ok := d.GetOk("cloud_brownfield"); ok {
+		cloudBrownfield = tmpVar.(string)
+		log.Printf("\n\n\n[DEBUG]cloudBrownfield in ctxCreate %v", cloudBrownfield)
+	}
+
```

Comment:
```suggestion
	cloudBrownfield := d.Get("cloud_brownfield").(string)
```

Reply:
cloud_brownfield might be None which cannot use .(string)